cli/cmd/wallet.go

```diff
@@ -35,9 +36,14 @@ var (
 
 			var output [][]string
 			for name, wallet := range cfg.Wallets.All {
+				kind := wallet.Kind
+				if alg, valid := wallet.Config["algorithm"]; valid {
```

Comment:
This is slightly ugly (as the config is wallet-specific). Could be added to the wallet factory instead and then you could run something like `wallet.PrettyKind()` that would instantiate the factory based on wallet kind and call `wf.PrettyKind(w.Config)`.